django-backend/fecfiler/web_services/dot_fec/dot_fec_serializer.py

```diff
@@ -43,6 +43,10 @@ def date_serializer(model_instance, field_name, mapping):
     return date.strftime("%Y%m%d") if date else ""
 
 
+def fixed_value_serializer(_model, _field, mapping):
```

Comment:
Is there any reason we aren't just storing pdf_attachment value in the database? That way, the value would come up with the API call and could be referenced using a standard/field mapping. 

It seems it could be simpler to and would also add traceability/debug if someone claimed they attached one but didn't or visa-versa.